lib/DxcSupport/HLSLOptions.cpp

```diff
@@ -206,6 +206,59 @@ static bool GetTargetVersionFromString(llvm::StringRef ref, unsigned *major, uns
   }
 }
 
+// SPIRV Change Starts
+#ifdef ENABLE_SPIRV_CODEGEN
+/// Checks and collects the arguments for -fvk-{b|s|t|u}-shift into *shifts.
+static bool handleVkShiftArgs(const InputArgList &args, OptSpecifier id,
```

Comment:
why does this need to be `static`  ?

Reply:
To keep this function local to this file.